app/models/tour_list_recording.rb

```diff
@@ -0,0 +1,16 @@
+class TourListRecording
+  attr_reader :contents
+
+  def initialize(initial_contents)
+    @contents = initial_contents || Hash.new(0)
+    @contents.default = 0
+  end
+
+  def add_recording(recording)
+    @contents[recording.id.to_s] = recording.landmark_id
```

Comment:
What's the point of this key-value pair? The recording inherently belongs to a landmark already.

Reply:
This key-value pair belongs to the "user can add their own recording to a landmark on their tour list" functionality. This hash is a check to make sure all landmarks on a tour list have a recording.